GOVERNANCE.MD

```diff
@@ -1,6 +1,91 @@
+
+Section 3. Board’s Role in Setting Node.js Foundation’s Strategic Direction.
+
+The Board will set the overall CC Policy. The policy will describe the overarching scope of the Node.js Foundation initiative, Node.js Foundation’s community vision, direction and expectations in the form of intent. The Board will use the CC as a delegate body for community representation, advisement, and overseeing cultural project implementation, scope and direction while they remain within the scope and direction of the policies as described in the CC Policy document and approved by the Board.
+
+Section 4. Establishment of the Community Committee.
+
+CC memberships are not time-limited. There is no fixed size of the CC. However, the expected target is between 9 and 12, to ensure adequate coverage of important areas of community expertise, balanced with the ability to make decisions efficiently.
+
+There is no specific set of requirements or qualifications for CC membership beyond these rules. The CC may add additional members to the CC by a standard CC motion and vote. A CC member may be removed from the CC by voluntary resignation, or by a standard CC motion.
+
+Changes to CC membership should be posted in the agenda, and may be suggested as any other agenda item.
+
+No more than one-fourth of the CC members may be affiliated with the same employer or leadership of a community/ecosystem organization. If removal or resignation of a CC member, or a change of employment by a CC member, creates a situation where more than one-fourth of the CC membership shares an employer, then the situation must be immediately remedied by the resignation or removal of one or more CC members affiliated with the over-represented employer(s).
+
+The CC members shall consist of active members of Community Projects as defined in Section 7.
+
+The CC shall meet regularly using tools that enable participation by the community (e.g. weekly on a Google Hangout On Air, or through any other appropriate means selected by the CC). The meeting shall be directed by the CC Chairperson. Minutes or an appropriate recording shall be taken and made available to the community through accessible public postings.
+
+Section 5. Responsibilities of the CC.
+
+Subject to such policies as may be set by the Board, the CC is responsible for all cultural development and outreach within the Node.js Foundation, including:
+
+- Outreach to community organizations
+- Documentation of community organizations
+- Cultural direction of Node.js Foundation
+- Overseeing Inclusivity WG
+- Project governance and process (including this policy)
+- Recommendations for building and developing community projects that align with needs of Node.js
+- Mediating cultural conflicts between Foundation and/or community projects
+- The CC will serve as Node.js Foundation’s primary community liaison body with external open source projects, consortiums and groups.
```

Comment:
To be consistent with the other bullet points, should we remove the "The CC will" prefix so that it reads as "Serve as Node.js Foundations's..."?